rest/alert_test.go

```diff
@@ -0,0 +1,562 @@
+package rest_test
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/ns1/ns1-go.v2/mockns1"
+	"gopkg.in/ns1/ns1-go.v2/rest/model/alerting"
+
+	api "gopkg.in/ns1/ns1-go.v2/rest"
+)
+
+func strPtr(str string) *string {
```

Comment:
Annoying that such a function is needed. I should add this to my "Why I Hate Go" presentation!